artio-codecs/src/test/java/uk/co/real_logic/artio/fields/DecimalFloatToStringTest.java

```diff
@@ -48,6 +48,8 @@ public void canEncodeDecimalFloat()
     {
         final DecimalFloat price = new DecimalFloat(value, scale);
 
-        assertEquals(input, price.toString());
+        // since toString of DecimalFloat does not return leading zeros, it fails
+        //assertEquals(input, price.toString());
```

Comment:
Why does this PR just comment out asserts? The the output of the `toString()` method should be consistent with printing the number on the `AsciiBuffer`.

Reply:
I left a comment so you could see it.  :-)
**must** it be consistent?
look at line below I use Float.valueOf.

Reply:
I think it should be consistent, you don't want to see a value in your debugger that is different from the one printed out on the buffer. The implementation of toString() does not have to be fast though as people should never use that on the critical path of their application.

Reply:
i tried to make it consistent, than encountered other test that fails...
test class **DecimalFloatToAndFromDoubleTest** method **shouldConvertFromDoubleAccurately**

> uk.co.real_logic.artio.fields.DecimalFloatToAndFromDoubleTest > shouldConvertFromDoubleAccurately FAILED
>     org.junit.ComparisonFailure: expected:<[].01> but was:<[0].01>
>         at org.junit.Assert.assertEquals(Assert.java:115)
>         at org.junit.Assert.assertEquals(Assert.java:144)
>         at uk.co.real_logic.artio.fields.DecimalFloatToAndFromDoubleTest.shouldConvertFromDoubleAccurately(DecimalFloatToAndFromDoubleTest.java:45)

as I understand the code, the consistency between double/float of java lang and DecimalFloat is **more important** than consistency of DecimalFloat to MutableAsciiBuffer.
if that is correct the test of DecimalFloatToStringTest.java might be obsolete.

Reply:
ok made it consistent. they both output with leading zeros.
had to modify test class **DecimalFloatToAndFromDoubleTest** accordingly.

Reply:
Thanks.